synapse/api/room_versions.py

```diff
@@ -193,6 +193,19 @@ class RoomVersions:
         msc3083_join_rules=False,
         msc2403_knocking=True,
     )
+    V8 = RoomVersion(
```

Comment:
Or maybe we should just update the `MSC3083` room version?

Reply:
Seems like for both room version 6 & 7 we removed the experimental ones at the same time.